frontend/workflows-lib/lib/utils/validationUtils.ts

```diff
@@ -0,0 +1,102 @@
+import { ScanRange } from "../types";
+
+export const isStrictPositiveInteger = (value: string): boolean =>
+  /^\d+$/.test(value) && Number(value) >= 0;
+
+export const parseExcludedScans = (input: string): number[] => {
+  const cleaned = input.replace(/[[\]]/g, "");
```

Comment:
Why do we accept square brackets but then just strip them out? This seems to add no utility for the user?

Reply:
This is to make it compatible with what some beamlines already expect when entering these kind of values.